test/user-test.js

```diff
@@ -0,0 +1,38 @@
+import { assert } from 'chai'
+import { sampleRecipeData, sampleUsersData } from '../src/data/sampleData';
+import {addRecipeToCook} from '../src/users'
+
+describe('users', () => {
+  const saige = sampleUsersData[0];
+  const ephriam = sampleUsersData[1];
+  const cookies = sampleRecipeData[0];
+  const porkChops = sampleRecipeData[1];
+  it('should be a function', () => {
+    assert.isFunction(addRecipeToCook)
+  })
+  it('should be able to add recipes to cook', () => {
+    let newSaige = addRecipeToCook(saige, cookies); 
+    let newEphriam = addRecipeToCook(ephriam, porkChops);
```

Comment:
I was able to implement this. The only it block that will have data creation within it will be the check for duplicates block which will reassign some data within the block in order to implement the test.